bears/vcs/git/GitCommitBear.py

```diff
@@ -291,6 +291,9 @@ def check_issue_reference(self, body,
         if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
             return
 
+        if not re.search(self.SUPPORTED_HOST_KEYWORD_REGEX[host], body):
+            return
```

Comment:
actually there are repos which require always an issue reference, or a JIRA ticket reference. So enforcing this on all commits is not completely unuseful.

alright behaviour is okay, but would be good if you implement another flag `body_enforce_issue_reference` to enforce this for all commits. Then you can still keep the "old" behaviour.

Also I think you should modify the code here to implement this behaviour: https://github.com/coala/coala-bears/pull/1333/files/4ce4c848a53d0c9b2024da8b9cea4fedad84ace5#diff-e0a334e6d00dfb6b1fbebb7c515ca4f8R333
In fact you just have to remove this if / extend it by
```python
if body_enforce_issue_reference and if len(results) == 0:
    yield Result(...)
```